README.md

```diff
@@ -220,7 +220,9 @@ developed by the Open Security Foundation (OSF) and its contributors.
 [CVE]: https://cve.mitre.org/
 [OSVDB]: https://en.wikipedia.org/wiki/Open_Source_Vulnerability_Database
 [GHSA]: https://help.github.com/en/articles/about-maintainer-security-advisories
-[CVSSv2]: https://www.first.org/cvss/v2/guide
-[CVSSv3]: https://www.first.org/cvss/user-guide
+[CVSSv2.0]: https://www.first.org/cvss/v2/guide
+[CVSSv3.0]: https://www.first.org/cvss/v3.0/user-guide
+[CVSSv3.1]: https://www.first.org/cvss/v3.1/user-guide
```

Comment:
The README already uses `[CVSSv3]` link names, so either we keep the name or update the README to use `[CVSSv3.1]`.